ajax/libs/dna.js/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "dna.js",
+  "autoupdate": {
+   "source": "git",
+   "target": "git://github.com/dnajs/dna.js.git",
+   "fileMap": [
+    {
+      "basePath":  "",
```

Comment:
@karan-96 One more blank after `:`. Please remove it. You can run the fix tool `fixFormat.js` https://github.com/cdnjs/cdnjs/tree/master/tools to fix it. Thanks!